src/aiida_icon/calculations.py

```diff
@@ -28,6 +28,7 @@ def define(cls, spec: calcjob.CalcJobProcessSpec) -> None:  # type: ignore[overr
         spec.input("model_namelist", valid_type=orm.SinglefileData)
         spec.input("restart_file", valid_type=orm.RemoteData, required=False)
         spec.input("wrapper_script", valid_type=orm.SinglefileData, required=False)
+        spec.input("env_file", valid_type=orm.SinglefileData, required=False)
```

Comment:
I would like the input name to reflect that this will be run as a script. The name should also be descriptive on it's own, without knowing about Sirocco. Below is a suggestion, but feel free to come up with something better.

```suggestion
        spec.input("setup_script", valid_type=orm.SinglefileData, required=False)
```

Reply:
Maybe `env_setup_script`?

Reply:
I remember from our meeting that we said we also allow executing commands there, not just env variables. However, this could also be contained in the `wrapper_script`, I suppose? In the end, none of us apart from you have any experience running W&C simulations, so I'd fully follow what you say here, @leclairm.

Reply:
It's absolutely fine to source a file that executes commands and is not particularly C&W related. The commands will just be executed in the process where it was sourced from (`source run_env.sh`) instead of executed in a sub-process (`./run_env.sh`), which is what we need to have the environment set up when ICON runs.

Reply:
In that case it is indeed not a "script" in bash parlance.

Reply:
I like `env_setup` the most so far for the input name.